### PR TITLE
SISRP-34089 - Place link to training page for eForms: committees

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-34089

### CODE REVIEW

**Comment by davidlimcheng** [comment]
LGTM, @redconfetti

File: src/assets/templates/widgets/student_resources.html [hunk 1]
```diff
@@ -55,7 +55,13 @@
               <a data-cc-campus-solutions-link-directive="studentResources.higherDegreesCommitteeForm"
                 data-cc-campus-solutions-link-directive-cc-page-name="currentPage.name"
                 data-cc-campus-solutions-link-directive-cc-page-url="currentPage.url"
-              ></a>: <span>Advance to Candidacy, apply for the Qualifying Exam, or Change your Higher Education Degree Committee.</span>
+              ></a>: <span data-ng-bind="studentResources.higherDegreesCommitteeForm.title"></span>
+            </div>
+            <div class="cc-inline-block">
+              <i class="fa fa-info-circle cc-icon-list-bullet-grey"></i>
+              <a href="http://grad.berkeley.edu/sis/tutorial" title="How to use the High Degree Committees Form">
```
Comment (davidlimcheng) [inline src/assets/templates/widgets/student_resources.html]: Typo here @redconfetti "High Degree"

File: src/assets/templates/widgets/student_resources.html [hunk 2]
```diff
@@ -55,7 +55,13 @@
...
+                How to use the High Degree Committees Form
```
Comment (davidlimcheng) [inline src/assets/templates/widgets/student_resources.html]: Here as well

Comment (redconfetti) [inline src/assets/templates/widgets/student_resources.html]: @davidlimcheng  good catch

**Comment by davidlimcheng** [comment]
*Thumbs Up Emoji*

**Pull request merged**